scripts.py

```diff
@@ -225,89 +230,369 @@ def run(self, data_source_name, metadata_records, epub_directory, cover_director
             work.set_presentation_ready()
             self._db.commit()
 
-class CSVExportScript(Script):
+
+class StaticFeedScript(Script):
```

Comment:
I wanted both scripts to have the same expectations for creating / translating headers as categories.